src/app/util/util.c

```diff
@@ -1212,42 +1212,42 @@ EmberStatus emberAfEndpointEventControlSetActive(EmberEventControl * controls, u
     return EMBER_SUCCESS;
 }
 
-EmberStatus emberAfEndpointEventControlSetDelayMS(EmberEventControl * controls, uint8_t endpoint, uint32_t delayMs)
-{
-    uint8_t index = emberAfIndexFromEndpoint(endpoint);
-    if (index == 0xFF)
-    {
-        return EMBER_INVALID_ENDPOINT;
-    }
-    return emberAfEventControlSetDelayMS(&controls[index], delayMs);
-}
-
-EmberStatus emberAfEndpointEventControlSetDelayQS(EmberEventControl * controls, uint8_t endpoint, uint32_t delayQs)
-{
-    uint8_t index = emberAfIndexFromEndpoint(endpoint);
-    if (index == 0xFF)
-    {
-        return EMBER_INVALID_ENDPOINT;
-    }
-    return emberAfEventControlSetDelayQS(&controls[index], delayQs);
-}
-
-EmberStatus emberAfEndpointEventControlSetDelayMinutes(EmberEventControl * controls, uint8_t endpoint, uint16_t delayM)
-{
-    uint8_t index = emberAfIndexFromEndpoint(endpoint);
-    if (index == 0xFF)
-    {
-        return EMBER_INVALID_ENDPOINT;
-    }
-    return emberAfEventControlSetDelayMinutes(&controls[index], delayM);
-}
-
-bool emberAfIsThisMyEui64(EmberEUI64 eui64)
-{
-    EmberEUI64 myEui64;
-    emberAfGetEui64(myEui64);
-    return (0 == memcmp(eui64, myEui64, EUI64_SIZE) ? true : false);
-}
+// EmberStatus emberAfEndpointEventControlSetDelayMS(EmberEventControl * controls, uint8_t endpoint, uint32_t delayMs)
```

Comment:
These functions will not be called and will block chip-server from linking, so comment them out.

Reply:
why do they block chip-server from linking?